app/views/apis/show.html.erb

```diff
@@ -26,3 +29,52 @@
 <% end %>
 
 <%= render partial: 'new_review_form', object: @review %>
+
+<script>
```

Comment:
Your script should rarely be embedded in the view unless you are doing something sensitive with keys that you can *only* do with embedded Ruby. This should be factored out into a javascript file that is compiled along with the rest of the asset pipeline for this app.